### ISSUE TITLE
Evaluation error

### ISSUE THREAD

**Issue by TheOtherMarcus** [terrastruct/TALA#49]
Trying to evaluate tala on Windows. d2 version 0.5.0, tala version 0.3.8. Get this error. Elk works fine.

C:\>d2 -l tala -s test.d2.txt
********************************
WARNING: THIS COPY OF TALA IS UNLICENSED AND IS FOR EVALUATION PURPOSES ONLY

missing api token: could not find $TSTRUCT_TOKEN or api_token in auth file ~\.config\tstruct\auth.json

Run d2plugin-tala.exe --help for more info.
********************************
********************************
WARNING: THIS COPY OF TALA IS UNLICENSED AND IS FOR EVALUATION PURPOSES ONLY

missing api token: could not find $TSTRUCT_TOKEN or api_token in auth file ~\.config\tstruct\auth.json

Run d2plugin-tala.exe --help for more info.
********************************
err: failed to compile: failed to unmarshal json: json: cannot unmarshal object into Go struct field Attributes.attributes.constraint of type []string

I can fake a token and then it says

failed to verify $TSTRUCT_TOKEN: illegal base64 data at input byte 4

but the final error is the same.

**Comment by dired** [comment]
I ran into the exact same issue (two same warnings, "unmarshal json") and fixed it just now by uninstalling, downloading the versions before the latest release (https://github.com/terrastruct/d2/releases/tag/v0.4.2, https://github.com/terrastruct/TALA/releases/tag/v0.3.7), and linking the binaries to my path (~/.local/bin/).

Of course you wanted to try the newest versions - I did not try linking the newest binaries. So I don't know if the error is maybe just with the installer and the newest binaries work or if we actually can only revert to the prior version of either tala or d2 (did not try) at this point.
